### PR TITLE
Tutorial Updated by @emmanueljet

### PR DESCRIPTION
**Pull request by emmanuelJet**
TUTORIAL: How to Get Started With Unit Testing Using forgae and How to Register and Update a Name with the SDK in Javascript updated by @emmanuelJet. @thepiwo, @alekseytsekov, and @Pegahbit kindly review #41.

AE Address: ak_rLoCtHE3NK9dKyCNonJFYWkEEfeAsDUWa887GsCKqV1rhSuT6

### CODE REVIEW

**Comment by emmanuelJet** [comment]
Changed `function` keyword to `entrypoint` in **Deploying Sophia smart contracts with init parameters** and How to **Get Started With Unit Testing Using forgae** tutorial.

**Comment by Jkat** [comment]
Do you have the output for the `aensUpdate` update call? Seems to be broken in the current state.

Also the rebase looks a bit strange to me since it's reverting back to some old mistakes.

File: deploy-with-init-params.md [hunk 1]
```diff
@@ -39,111 +37,95 @@ forgae node
...
 
 const deploy = async (network, privateKey, compiler) => {
     let deployer = new Deployer(network, privateKey, compiler)
 
-	let contract = await deployer.deploy("./contracts/ExampleContract.aes", [42])
+    let contract = await deployer.deploy("./contracts/ExampleContract.aes")
+
+    // Getting savedString value in our ExampleContract
+    let get_string = await contract.call('get_string')
+    console.log(get_string.value)
```
Comment (alekseytsekov) [inline deploy-with-init-params.md]: with current forgae, syntax is `await contract.decode()`, isn't it? and with the latest js-sdk is `decodedValue` ?

File: get-started-with-unit-testing.md [hunk 2]
```diff
@@ -1,138 +1,197 @@
...
-	keypair: wallets[0],
-	nativeMode: true,
-	networkId: 'ae_devnet',
-	compilerUrl: 'http://localhost:3080'
-
+  url: config.host,
+  internalUrl: config.internalHost,
+  keypair: wallets[0],
+  nativeMode: true,
+  networkId: 'ae_uat'
```
Comment (alekseytsekov) [inline get-started-with-unit-testing.md]: why did you remove `compilerUrl`

File: get-started-with-unit-testing.md [hunk 3]
```diff
@@ -1,138 +1,197 @@
...
-        deployer = new Deployer('local', ownerKeyPair.secretKey)
-    })
+    // Getting savedString value in our ExampleContract
+    let get_string = await contract.call('get_string')
+    console.log(get_string.value)
 
-    it('Deploying Example Contract', async () => {
-        const deployPromise = deployer.deploy(EXAMPLE_CONTRACT_PATH) // Deploy it
+    // Writing new value ('hello world') to our saved_string
+    await contractCall('register_string', 'hello world')
```
Comment (alekseytsekov) [inline get-started-with-unit-testing.md]: typo-> await contract.call...

**Comment by emmanuelJet** [comment]
Errors fixed
